pnpm-workspace.yaml

```
@@ -1,6 +1,6 @@
 packages:
   - 'apps/**'
   - 'libs/**'
-  - '!**/libs/@guardian/source/*/**'
+  - '!**/libs/@guardian/*/*/**'
```

Should this be narrower so that other packages are not accidentally not included. Or will we never want packages at this level?
```suggestion
  - '!**/libs/@guardian/source/*/**'
  - '!**/libs/@guardian/source-development-kitchen/*/**'
```